app/components/arclight/collection_info_component.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Arclight
+  # Render information about the collection
+  class CollectionInfoComponent < ViewComponent::Base
+    def initialize(collection:)
+      super
+
+      @collection = collection
+    end
+
+    attr_reader :collection
+
+    delegate :total_component_count, :last_indexed, :collection_unitid, to: :collection
+
+    def online_item_count
+      Arclight::OnlineItemCount.for(collection: collection)
```

Comment:
Can we avoid the extra solr query and either index this too, or get Solr to calculate it as part of the collections subquery?

Reply:
Yes, but I'm not sure how one would do that.

Reply:
I think the existing `has_online_content_ssim` probably isn't what we want, now that I look at it -- it applies to any element in the hierarchy with a `dao` node, not just the component that actually has a DAO.

I wonder if this is supposed to count components (e.g. the c + c1...c12 nodes that contain a `did/dao`) or the actual DAOs.

Reply:
From Gary:

> My thought is that “Total components” is the count of component objects, not factoring in how many digital objects they might have, and “Items available online” is the total count of digital objects in the collection, not factoring in how many components there are.

So I think we have to index a new value at the collection level (the count of `//dao` elements?), and can just use that.